sdk/identity/azure-identity/src/private/managed_identity_source.hpp

```diff
@@ -43,8 +44,8 @@ namespace Azure { namespace Identity { namespace _detail {
     {
     }
 
-    std::string const& GetClientId() const { return m_clientId; }
-    std::string const& GetAuthorityHost() const { return m_authorityHost; }
+    // std::string const& GetClientId() const { return m_clientId; }
+    // std::string const& GetAuthorityHost() const { return m_authorityHost; }
```

Comment:
Is this intentional?

Reply:
Nope, good catch. Although I do believe we don't use them anywhere (which is what I was testing by removing these).